components/Slider/src/MDCSlider.m

```diff
@@ -229,9 +250,22 @@ - (void)updateColorsForState {
   _thumbTrack.trackOffTickColor = [self backgroundTrackTickColorForState:self.state];
 }
 
+- (void)updateThumbForState {
+  if (!self.isStatefulAPIEnabled) {
+    return;
+  }
+
+  _thumbTrack.thumbRadius = (CGFloat)[self thumbRadiusForState:self.state];
+}
+
 #pragma mark - ThumbTrack passthrough methods
 
 - (void)setThumbRadius:(CGFloat)thumbRadius {
+  if (self.statefulAPIEnabled) {
+    return;
+  }
+  double radius = (double)thumbRadius;
+  _thumbRadiusesForState[@(UIControlStateNormal)] = [NSNumber numberWithDouble:radius];
```

Comment:
I don't think that using the old API should affect the new API.  What do you think?  Probably easiest to just assign the `_thumbTrack.thumbRadius`  if the new API is disabled.